ahk/_hotkey.py

```diff
@@ -170,6 +203,13 @@ def start(self) -> None:
         dispatcher_thread.start()
 
     def stop(self) -> None:
+        assert self._running is True, 'Not running! Must be started first!'
+        assert self._dispatcher_thread is not None
+        for i in range(1, 6):
+            if self._proc is not None:
+                break
+            logging.debug(f'stop called before dispatched has started proc. Waiting for proc ({i}/5)')
+            time.sleep(0.2)
```

Comment:
Sleeping for a shorter period would likely be better for performance.

```suggestion
        for i in range(1, 11):
            if self._proc is not None:
                break
            logging.debug(f'stop called before dispatched has started proc. Waiting for proc ({i}/10)')
            time.sleep(0.1)
```